lib/validators/specResolver.ts

```diff
@@ -218,6 +220,10 @@ export class SpecResolver {
         this.resolveNullableTypes()
       }
     } catch (err) {
+      // to avoid double wrap the exception
+      if (typeof err === "object" && err.id && err.message) {
```

Comment:
It needs to wrap the error message as `internal error`.

Reply:
Because sometimes  the error message  have already been wrapped as internal err

Reply:
I know, but does all the error object here is `internal` err? If not, we still need to wrap them.

Reply:
if the exception have already specify the id and message , It must be a exception that has been wrapped
so here we don't need to wrap it again